### PR TITLE
Bare minimum for custom effects (take 2)

### PR DESCRIPTION
**Pull request by kaste**
Same as #3460. Removed prestored effects in createAccessors as suggested per https://github.com/Polymer/polymer/pull/3460#issuecomment-206599307.

### CODE REVIEW

**Comment by kaste** [comment]
The last commit here is a minor refactoring in Templatizer that fits on top of your https://github.com/Polymer/polymer/pull/3573/commits/59b5f240dadc61e16fedc98c17dbd9c1c17861af

File: src/standard/notify-path.html [hunk 1]
```diff
@@ -274,10 +274,12 @@
         var fx$ = this._propertyEffects && this._propertyEffects[model];
         if (fx$) {
           for (var i=0, fx; (i<fx$.length) && (fx=fx$[i]); i++) {
```
Comment (kaste) [inline src/standard/notify-path.html]: Note to self: Must prestore length just like in accessors!

File: src/lib/bind/accessors.html [hunk 2]
```diff
@@ -224,7 +228,7 @@
           // queued events during configuration can theoretically lead to
           // divergence of the passed value from the current value, but we
           // really need to track down a specific case where this happens.
-          value = target[property];
+          value = target.__data__ ? target.__data__[property] : target[property];
```
Comment (fearphage) [inline src/lib/bind/accessors.html]: I believe this reads a little clearer. 

```js
value = (target.__data__ || target)[property]
```

**Comment by TimvdLippe** [comment]
Closing per https://github.com/Polymer/polymer/pull/3460#issuecomment-320333064

**Pull request closed without merging**